packages/multi-select-combo-box/src/vaadin-multi-select-combo-box.js

```diff
@@ -513,16 +588,36 @@ class MultiSelectComboBox extends InputControlMixin(ThemableMixin(ElementMixin(P
       return;
     }
 
-    this._chips.forEach((chip) => {
+    // Clear all chips except the overflow
+    const chips = Array.from(this._chips).reverse();
+    const overflow = chips.pop();
+
+    chips.forEach((chip) => {
       chip.remove();
     });
 
     const items = [...this.selectedItems];
 
+    let refNode = overflow.nextElementSibling;
+
+    // Input flex-basis (4em = 64px)
+    const INPUT_MIN_WIDTH = 64;
+    const CHIP_MIN_WIDTH = 60;
```

Comment:
I am thinking about these values and how the chips will work if the component font-size is different from `16px`?

Reply:
I’m happy Diego was brave enough to raise this topic. I was also wondering, but didn't want to make a fuss about it 😄

Reply:
I agree that this needs to be changed. How about using custom CSS properties? Something like this: 

- `--_vaadin-multi-select-combo-box-chip-min-width`
- `--_vaadin-multi-select-combo-box-input-min-width`

We could then use the latter also as a flex-basis for the `<input>` instead of hardcoding `4em` there.

Reply:
Updated to use custom CSS properties instead of hardcoded numbers.